lib/releaselib.sh

```diff
@@ -52,7 +52,7 @@ release::get_job_cache () {
   done < <(jq -r '.[] | select(.result == "SUCCESS") | select(.version != "") | [.version,.buildnumber] | "\(.[0]) \(.[1])"' $tempjson |\
    LC_ALL=C sort -rn -k2,2) |\
   while read version buildnumber; do
-    echo "JOB[$buildnumber]=$version"
+    [[ -n $buildnumber && -n $version ]] && echo "JOB[$buildnumber]=$version"
```

Comment:
yeah. since the test-infra bug was fixed early last week, we should have plenty of runs since then we can consider instead. fixing the bad data probably isn't worth the effort, and it should roll off eventually.